tests/test_predicates.py

```diff
@@ -144,6 +105,6 @@ def test_lift_matrix(self):
         self.assertTrue(equal(target, lift_matrix(base, list_test)))
         self.assertTrue(equal(target, lift_matrix(base, vector_test)))
         self.assertTrue(equal(target, lift_matrix(base, matrix_test)))
-    '''
```

Comment:
Why did you uncomment this but not change it so it works with the new `Point`, `Vector`, and `Matrix`?